js/src/vue/CustomObject/FieldPreview/Sidebar.vue

```diff
@@ -0,0 +1,77 @@
+<script setup>
+    import Field from "./Field.vue";
+    import {computed, ref, onMounted} from "vue";
+
+    const props = defineProps({
+        inactive_fields: Map,
+        add_edit_fn: String,
+    });
+
+    const search = ref('');
+
+    const unused_native_fields = computed(() => {
+        return new Map([...props.inactive_fields].filter(([key, field]) => (field.customfields_id ?? -1) < 0));
+    });
+    const unused_custom_fields = computed(() => {
+        return new Map([...props.inactive_fields].filter(([key, field]) => (field.customfields_id ?? -1) >= 0));
+    });
+
+    function getMatched(fields) {
+        if (search.value === '') {
+            return fields;
+        }
+        const results = new Map();
+        for (const [key, field] of fields) {
+            if (field.label.toLowerCase().includes(search.value.toLowerCase())) {
+                results.set(key, field);
+            }
+        }
+        return results;
+    }
+
+    onMounted(() => {
+        $('.fields-sidebar .new-custom-field').on('click', () => {
+            window[props.add_edit_fn](-1);
+        });
+    });
+
+</script>
+
+<template>
+    <div class="h-100 d-flex col-auto flex-column p-0 ps-2 fields-sidebar">
+        <span class="fs-2">{{ __('Add more fields') }}</span>
+        <input type="text" class="form-control mb-3" name="search" :placeholder="__('Search')" v-model="search" />
```

Comment:
```suggestion
        <input type="text" class="form-control mb-3" name="search" :placeholder="{{ __('Search') }}" v-model="search" />
```

Reply:
This is Vue. The `{{ }}` syntax is used for content, but not for attribute values.
https://vuejs.org/guide/essentials/template-syntax#attribute-bindings